src/psij/launchers/scripts/mpi_launch.sh

```diff
@@ -5,10 +5,19 @@ source $(dirname "$0")/launcher_lib.sh
 _PSI_J_PROCESS_COUNT="$1"
 shift
 
+IS_OPENMPI=0
+if mpirun -version | grep "Open MZI" >/dev/null 2>&1; then
+    IS_OPENMPI=1
+fi
+
 pre_launch
 
 set +e
-mpirun -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+if "$IS_OPENMPI" == "1"; then
+    mpirun --oversubscribe -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+else
+    mpirun -n $_PSI_J_PROCESS_COUNT "$@" 1>$_PSI_J_STDOUT 2>$_PSI_J_STDERR <$_PSI_J_STDIN
+fi
```

Comment:
Once the number of cases increase (and they will), we should reconsider the code replication.  For now that works.